examples/nextjs/next.config.js

```diff
@@ -1,13 +1,17 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {}
 
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true'
+})
+
 const pkg = require('./package.json')
 // * Only required to make it work with the monorepo. Is not required otherwise
 const withTM = require('next-transpile-modules')(
   // * All references to workspace packages are transpiled
   Object.entries(pkg.dependencies)
-    .filter(([name, version]) => version.startsWith('workspace'))
```

Comment:
if we decide to use `workspace:xyz` instead of `workspace:*` we will need to update this code.
I would keep the test on `workspace` without the wildcard as there's probably no other scenario than pnpm where a version would start with `workspace`